apps/rush-lib/src/schemas/version-policies.schema.json

```diff
@@ -83,10 +83,14 @@
         "mainProject": {
           "description": "The main project for this version policy",
           "type": "string"
+        },
+        "shouldSkipChangelogVerification": {
```

Comment:
We should talk about this name. What do you think about `noChangeLogRequired` instead?

Is the behavior here just to not error when `rush change -v` is run, or is it to not prompt for changelogs at all?

Reply:
The latter. NoChangeLogRequired works for me